src/Resources/translations/messages.en.yml

```diff
@@ -2,6 +2,7 @@ global.changesSaved: Your changes have been saved.
 navbar.login: Login
 navbar.myAccount: My Account
 base.footer.contactUs: Contact us
+base.footer.companyRegister: Become a partner
 base.footer.help: Help
 base.footer.legal: Legal notice
 base.footer.suggest: Suggest a restaurant
```

Comment:
Can you also add the `form.address.company*` keys to `messages.en.yml` please? The `en` language is the fallback/default, so translations should be present here so that they can be crowd-translated [here](http://translate.coopcycle.org/)